criteria.tex

```diff
@@ -1,10 +1,10 @@
 \section{Criteria to begin the LSST}
 
-Armed with an understanding of the system performance as outlined above and the Operations team readiness, we will use a set of objective criteria to gate the start of the LSST. These criteria will be concise and easily understandable so that the community of scientists and stakeholders counting on Rubin and the survey will have confidence the Observatory is on track.
+Armed with an understanding of the \it{as--built} system performance as outlined above and the Operations team readiness, we will use a set of objective criteria to gate the start of the LSST. These criteria will be concise and easily understandable so that the community of scientistsm, rubin staff and other stakeholders counting on Rubin and the survey can have confidence the Observatory is on track.
 
-The criteria we will use that are listed below are only to start the survey. There may be key processes within the overall system, including data management and processing that require further work after handover (beyond the construction project requirements), but unless they would prevent delaying the survey start, they are not discussed or enumerated here.
+The criteria articulated below are meant only to provide gating guidance signifying the survey start. There will likely be key processes within the overall system, including data management and processing where further work for improvments are required or desired after handover (beyond the construction project requirements). Unless these would prevent yjr acquisition and saving of science qquality data, thereby delaying the survey start, they are not discussed or enumerated here.  These and other criteria (TBD) will be used to gage and moniter the overall performance of the Observatory and progress towards the 10-year survey objectives going forware, with nominal $T=0$ defined by when start criteria described herein are met.
 
-The Operations team discussed an initial set of criteria with the Science Advisory Committee and community at the 2024 Rubin Community Workshop. The current criteria have been evolved since that meeting and are shown in the table below.
+The initial set of criteria developed by Operations team were discussed with the Science Advisory Committee and community at the 2024 Rubin Community Workshop. These criteria have evolved since that meeting abd are presented here in the table below.
```

Comment:
```suggestion
The initial set of criteria developed by Operations team were discussed with the Science Advisory Committee and community at the 2024 Rubin Community Workshop. These criteria have evolved since that meeting and are presented here in the table below.
```